packages/db/src/resources/bytecodes.ts

```diff
@@ -42,7 +42,7 @@ export const bytecodes: Definition<"bytecodes"> = {
 
     input BytecodeInput {
       bytes: Bytes!
-      linkReferences: [LinkReferenceInput]
+      linkReferences: [LinkReferenceInput!]
```

Comment:
Ditto here, thought this could be empty.

Reply:
Ditto above :)